Documentation/devel/stage1-implementors-guide.md

```diff
@@ -28,32 +28,31 @@ This allows users to employ different containment strategies on the same host ru
 Entrypoints
 -----------
 
-### `rkt run` => "coreos.com/rkt/stage1/run"
+### `rkt run` => `coreos.com/rkt/stage1/run`
 
-0. rkt prepares the pod's stage 1 and stage 2 images and Pod Manifest under "/var/lib/rkt/pods/$uuid", acquiring an exclusive advisory lock on the directory
-1. chdirs to "/var/lib/rkt/pods/$uuid"
-2. resolves the "coreos.com/rkt/stage1/run" entrypoint via Annotations found within "/var/lib/rkt/pods/$uuid/stage1/manifest"
-3. executes the resolved entrypoint relative to "/var/lib/rkt/pods/$uuid/stage1/rootfs"
+0. rkt prepares the pod's stage 1 and stage 2 images and Pod Manifest under `/var/lib/rkt/pods/$uuid`, acquiring an exclusive advisory lock on the directory
```

Comment:
Also correct the path to /var/lib/rkt/pods/**run**/$uuid/... here and next three lines (Unless my /var/lib/rkt/ layout is wrong)